OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -30,7 +31,8 @@ static CaptureOptions::InstrumentedFunction::FunctionType IntrumentedFunctionTyp
 }
 
 ErrorMessageOr<void> CaptureClient::StartCapture(
-    ThreadPool* thread_pool, int32_t pid,
+    ThreadPool* thread_pool, int32_t process_id, std::string process_name,
+    std::shared_ptr<Process> process,
     absl::flat_hash_map<uint64_t, FunctionInfo> selected_functions) {
```

Comment:
Maybe this is the point to introduce CaptureParameters class, also considering there is soon going to be list of tracepoint and process will get replaced.

CaptureClient only forwards them to OnCaptureStarted anyways.

Reply:
Yeah, maybe. Still not 100% sure, if I should add all of those, or keep it as it was, and reconstruct the others on the OnCaptureStarted. I guess the only issue was, that there was would be different threads reading/writing involved.

Reply:
Lets do this in a sep. PR.